spacy/language.py

```diff
@@ -2328,13 +2328,18 @@ def restore(self) -> None:
         self[:] = []
 
 
-def _copy_examples(examples: Iterable[Example]) -> List[Example]:
+def _copy_examples(
+    examples: Iterable[Example], copy_x: bool = True, copy_y: bool = False
```

Comment:
Can we make the new parameters keyword-only (if only to eliminate one potential source of confusion)?

Reply:
Dones.